dataprofiler/profilers/base_column_profilers.py

```diff
@@ -11,9 +11,8 @@
 import numpy as np
 import pandas as pd
 
-from dataprofiler.profilers.profiler_options import BaseInspectorOptions
-
 from . import utils
+from .profiler_options import BaseInspectorOptions, BaseOption
 
 BaseColumnProfilerT = TypeVar("BaseColumnProfilerT", bound="BaseColumnProfiler")
```

Comment:
actually concerned there may be an issue with the rebase here.... I thought there was or should be another small class here... but don't see it in `main`, `dev`, or `feature/profile-serialization`. Just take your time reviewing

Reply:
disregard -- the thing I thought would be here is actually properly in `[order_column_profile.py](https://github.com/capitalone/DataProfiler/pull/908/files/f33b3fa4692098e9f08ac755d2c132a3b4245aea#diff-2b7e9460d75358fbc35b3ad6ae76b150dbf02459bad20123b5d7ee4d3871c886)`